packages/react/src/autocomplete-trigger/AutocompleteTrigger.tsx

```diff
@@ -2,32 +2,61 @@ import { PopperAnchor } from "@radix-ui/react-popper";
 import { type ComponentPropsWithRef, forwardRef } from "react";
 
 import { useAutocompleteContext } from "../autocomplete-context";
-import { SearchInput } from "../search-input";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconAngleDown } from "../icons/IconAngleDown";
+import { IconX } from "../icons/IconX";
+import { Input } from "../input";
 import { extractSprinkles } from "../sprinkles";
 
-type AutocompleteTriggerProps = ComponentPropsWithRef<typeof SearchInput>;
+type AutocompleteTriggerProps = {
+  onValueClear?: () => void;
```

Comment:
i don't think we need to support this onValueClear prop? autocomplete already has `value` and `onValueChange` props